pulp_puppet_plugins/pulp_puppet/plugins/importers/directory.py

```diff
@@ -262,8 +262,8 @@ def _import_modules(self, module_paths):
             _logger.debug(IMPORT_MODULE, dict(mod=module_path))
 
             module.set_storage_path(os.path.basename(module_path))
```

Comment:
I figured the call to `set_storage_path` is not needed anymore because import_content specifies the `module_path`. Why is `module_path` being used twice?